terra-app-setup-chart/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v2
 name: terra-app-setup
 description: Chart for set up entities for deploying Terra applications
 type: application
-version: 0.0.9
+version: 0.0.10
```

Comment:
Probably want to coordinate with this PR depending on who merges first https://github.com/DataBiosphere/terra-app/pull/37/files